submissions/password/README.md

```diff
@@ -0,0 +1,5 @@
+This is a simple demonstration of how the string works in python.
+The demonstration lets to create a password and then asks you for the password when it loads.
+
+
+if you want to run in terminal type "python3 password/password.py"
```

Comment:
Missing the following in the README:
- List of contributors
- Why you contributed it